keyboards/wolf/ts60/ts60.h

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 <contact@vwolf.be>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+/* TS60 Keymap Definitions */
+#define LAYOUT_60_ansi( \
+    K00, K01, K02, K03, K04, K05, K06, K07, \
+    K10, K11, K12, K13, K14, K15,      K17, \
+    K20, K21, K22, K23, K24, K25, K26, K27, \
+         K31, K32, K33, K34, K35,      K37, \
+    K40, K41, K42, K43, K44, K45, K46, K47, \
+         K51, K52, K53, K54, K55,      K57, \
+    K60, K61, K62, K63, K64, K65, K66, K67, \
+    K70, K71, K72, K73, K74, K75,      K77, \
+    K80, K81, K82, K83, K84, K85, K86, K87, \
+    K90,                     K95,      K97 \
```

Comment:
Specifically, this needs to be rearranged so it looks like the actual key layout of the keyboard.  Eg, something like this: 

https://github.com/qmk/qmk_firmware/blob/9afc64cd61da5e3e1d2fd72a83602709a6e4e4b8/keyboards/dz60/dz60.h#L226-L245

And so that the keymaps can use sommething like: 
```c
    [0] = LAYOUT_60_ansi(
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL
    )

```

Reply:
oh I see, but the second part is the actual layout in the matrix?

Reply:
Yup, the second part is the actual wired up matrix.  The top part doesn't need to match, as the LAYOUT part allows you to remap it.

Reply:
Ah I see, I'll get on it.

Reply:
Sorry for the delay, but it should be all updated